include/fs/fs.h

```diff
@@ -192,6 +192,21 @@ struct fs_statvfs {
  * @}
  */
 
+/**
+ * @brief The name under which a zephyr,fstab entry mount structure is
+ * defined.
+ */
+#define FS_FSTAB_ENTRY(nodeid) _CONCAT(z_fsmp_, nodeid)
```

Comment:
We usually use `node_id`, so if you wouldn't mind changing to that here and elsewhere in this PR, I think it would make for nice consistency throughout the tree.